### PR TITLE
[WIP] Move documentation to be generated by couscous

### PR DESCRIPTION
**Pull request by malukenho**
- [x] Move existent pages to be generated using couscous
- [x] Create extras pages in `md` to be converted to markdown
- [x] Script to generate documentation
- [x] Link documentation versions

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,8 +6,16 @@ php:
   - hhvm
   - hhvm-nightly
 
+env:
+  global:
+    - GIT_NAME:  "'Marco Pivetta'"
+    - GIT_EMAIL: ocramius@gmail.com
+    - GH_REF:    github.com/Ocramius/ProxyManager
+    - secure:    "KEY"
```
Comment (malukenho) [inline .travis.yml]: @Ocramius I need a valid key to do the tests.

Comment (Ocramius) [inline .travis.yml]: @malukenho please point me to any documentation where this key should be generated

Comment (malukenho) [inline .travis.yml]: @Ocramius [http://couscous.io/docs/travis.html](http://couscous.io/docs/travis.html)

Comment (Ocramius) [inline .travis.yml]: @malukenho please rebase

File: .travis.yml [hunk 2]
```diff
@@ -24,7 +32,8 @@ matrix:
 
 after_script:
   - sh .travis.coverage.sh
+  - wget https://scrutinizer-ci.com/ocular.phar
```
Comment (Ocramius) [inline .travis.yml]: This is actually handled in `.travis.coverage.sh`

File: .travis.yml [hunk 3]
```diff
@@ -24,7 +32,8 @@ matrix:
...
+  - php ocular.phar code-coverage:upload --format=php-clover ./clover.xml
```
Comment (Ocramius) [inline .travis.yml]: Same as above

File: .travis.yml [hunk 4]
```diff
@@ -7,8 +7,16 @@ php:
...
+env:
+  global:
+    - GIT_NAME:  "'Marco Pivetta'"
+    - GIT_EMAIL: ocramius@gmail.com
+    - GH_REF:    github.com/Ocramius/ProxyManager
+    - secure: HUDB66EnmKFx2A+E3HxvuX/7XSjgfNy8dnzcpG+K9mf/JM5JXZflK3xO3J3LCgSQG6Ptwr2yy//7vX+PdJMdSRTmP+d/dy6v8yLRP+GvYZXLIbuKg7dQjLerOxJcy9JhU2eAEBevc002PwFqLalfhCyWLvLiCof8++E2uGbvTSU=
+
 before_script:
   - sh .travis.install.sh
+  - composer global require couscous/couscous
```
Comment (Ocramius) [inline .travis.yml]: Can we add it to the dev dependencies instead?

**Comment by Ocramius** [comment]
@mnapoli you know that this means that @malukenho will be bothering you a lot, right? :-)

**Comment by Ocramius** [comment]
creeping

**Comment by mnapoli** [comment]
haha no problem

**Pull request merged**